guides/source/active_record_callbacks.md

```diff
@@ -538,7 +595,21 @@ irb> @user.save # updating @user
 User was saved to database
 ```
 
+### Transactional callback ordering
+
+When defining multiple `after_commit` or `after_rollback` callbacks, the current behavior is they will be executed in reverse order. This may be surprising.
+
+```ruby
+class User < ActiveRecord::Base
+  after_commit { puts("this actually gets called second") }
+  after_commit { puts("this actually gets called first") }
+end
+
+NOTE: This applies to all `after_*_commit` variations too, such as `after_destroy_commit`.
+
+Hopefully this will be fixed in future versions of Rails, depending on if this [PR](https://github.com/rails/rails/pull/46992) gets merged.
```

Comment:
I don't love this wording, and it feels like a bit of false hope. Even if the PR gets merged, it has to be released and users must upgrade.

Reply:
@ghiculescu I've removed the reference for now, we can add it after your PR gets merged. :pray: ([e184684](https://github.com/rails/rails/pull/46996/commits/e18468424f639543b57cb7a55068dad48117e59f))